use-cases/inferencing/cost-optimization/gcsfuse/AchievingFasterPodStartup.md

```diff
@@ -0,0 +1,319 @@
+        gke-gcsfuse/cpu-limit: "0"
+        gke-gcsfuse/memory-limit: "0"
+        gke-gcsfuse/ephemeral-storage-limit: "0"
+      ...
+      - name: inference-server
+        ...
+        volumeMounts:
+        - name: gcsfuse
+          mountPath: /gcs/${MODEL_NAME}/${MODEL_VERSION}/
+          readOnly: true
+      volumes:
+      - name: gcsfuse
+        csi:
+          driver: gcsfuse.csi.storage.gke.io
+          volumeAttributes:
+            bucketName: ${GCS_BUCKET}
+            mountOptions: "metadata-cache:ttl-secs:-1,metadata-cache:stat-cache-max-size-mb:-1,metadata-cache:type-cache-max-size-mb:-1,metadata-cache:negative-ttl-secs:0,file-cache:max-size-mb:-1,file-cache:cache-file-for-range-read:true,file-cache:enable-parallel-downloads:true,implicit-dirs,only-dir:${MODEL_NAME}/${MODEL_VERSION}/"
+            skipCSIBucketAccessCheck: "true"
+       ...
+```
+
+**Note**: Smart _mountOptions_ defaults are set from GKE version v1.33. So, if
+you are deploying your workload in G2, A2, A3, A4, A4X, ct5l, ct5lp, ct5p, and
+ct6e machine families, all you need to do is to configure _only-dir_ parameter
+
+3. **Prefer RAM disk for Cloud Storage FUSE caching:**
+
+[GPU](https://cloud.google.com/compute/docs/accelerator-optimized-machines) and
+[TPU](https://cloud.google.com/tpu) machine families have a sizable amount of
+CPU memory that is commonly not fully utilized by inference servers, resulting
+in wasted resources. If that’s the case for you, you can leverage any unused RAM
+as a high-speed cache for GCSFuse. Below you can see how to configure GCSFuse to
+use RAM disk .
+
+**Note:** The Cloud Storage FUSE cache is persistent, meaning the RAM used to
+load the model is not recycled. So, if you can't afford to use RAM disk, we
+highly recommend you use Local SSDs. There is no extra setup needed to configure
+Cloud Storage FUSE to use Local SSDs on
+[A2](https://cloud.google.com/compute/docs/accelerator-optimized-machines#a2-vms),
+[A3](https://cloud.google.com/compute/docs/accelerator-optimized-machines#a3-vms),
+and
+[A4](https://cloud.google.com/compute/docs/accelerator-optimized-machines#a4-vms)
+GPU machine families, meaning you can skip the yaml snippet below. For the
+[G2](https://cloud.google.com/compute/docs/accelerator-optimized-machines#g2-vms)
+machine family,
+[you must attach Local SSDs during nodepool creation](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd#node-pool).
+For TPU machine families, there is no Local SSD option, so you must use the RAM
+disk approach below.
+
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vllm-openai-gcs-tuned-llama33-70b-a100
+spec:
+  replicas: 1
+    ...
+      volumes:
+      - name: gke-gcsfuse-cache
+        emptyDir:
+          medium: Memory
+      - name: gke-gcsfuse-tmp
+        emptyDir:
+          medium: Memory
+      - name: gke-gcsfuse-buffer
+        emptyDir:
+          medium: Memory
+```
+
+4. **Prefetch the LLM model:**
```

Comment:
Same comment as above: instead of numbered bold text, can it be a smaller heading?